middleware-catalog-module/.idea/aws.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This files inside .idea directory should not be here. They are on .ignore. You need to delete them in your machine then make a git add, git commit, git push.